packages/stable/src/api/documents/types.gen.ts

```diff
@@ -377,11 +429,10 @@ export interface DocumentsAggregateCardinalityValuesItem {
  */
 export type DocumentsAggregateCardinalityValuesRequest = DocumentSearchInAggregate & DocumentSearchFilter & {
     aggregate: "cardinalityValues";
+    aggregateFilter?: DocumentAggregateFilter;
     properties: {
         property: DocumentFilterProperty;
     }[];
-    limit?: number;
-    cursor?: string;
```

Comment:
@tommythorsen where these removed in our service?

Reply:
They were in the contract, but were not supported by the API, so I removed them from the contract

Reply:
yes